app/HELPER.MD

```diff
@@ -0,0 +1,9 @@
+> Some curl commands so no need to type and good for examples
+
+```sh
+# post recipes
+curl -i -X POST localhost:3000/v1/recipe -d '{}' -H 'content-type: application/json'
+
+# post recipe transaction
+ curl -i -X POST localhost:3000/v1/recipe/43283aa1-a065-41bd-bec1-d6cdc4362f5c/creation -d '{}' -H 'content-type: application/json'
+```
```

Comment:
Remove this file, its unnecessary as we provide API documentation.

Reply:
it was purely for me and maybe others could benefit as well. II'm not using the open api interface for testing something I'm working on.

File has been removed